.gitignore

```diff
@@ -20,3 +20,4 @@ commonjs/
 es/
 coverage/
 tmp/
+!examples/tmp
```

Comment:
isn't the exclamation mark to remove the /tmp from ignore, an exception?

Reply:
I can't remember the exact problem with that, but this is necessary for having the `/examples/tmp` folder, not ignored. In the GH workflow, the existing, deployed examples are checkout from the `gh-pages` to the default branch into the `/examples/tmp` directory. If this directory would be ignored, the checkout wouldn't work. Only newly created examples would be in the `/examples/tmp` which would be deployed, thus the `gh-pages` branch would be overwritten.

In the `code-snippets` repo the `.gitignore` looks the same and the `tmp` folder is visible to Git - https://github.com/handsontable/code-snippets/blob/master/.gitignore

Reply:
It's because in the previous line this repo ingores all `tmp/` folders. I was just wondering why do we need `tmp` folder in the repo. Wouldn't it be better to create it when needed? I'm not a fan of keeping empty folders, especially the temporary ones.

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/83e990b4affc9c6ec7cd3209a5d47e8bb249d0cc